<launch>
  <arg name="config_file"
    default="$(find vio_ekf)/launch/config/euroc.yaml"/>

    <node name="vio_ekf" pkg="vio_ekf" type="vio_ekf" output="screen">
    <!-- Calibration parameters -->
      <rosparam command="load" file="$(arg config_file)"/>

      <param name="grid_row" value="4"/>
      <param name="grid_col" value="5"/>
      <param name="per_grid_feature_num" value="2"/>
      <param name="time_shift" value="0.0"/>

      <remap from="~imu" to="/imu0"/>
      <remap from="~cam0_image" to="/cam0/image_raw"/>
      <remap from="~cam1_image" to="/cam1/image_raw"/>
    </node>

  <node pkg="rviz" type="rviz" name="rviz"
      args="-d $(find vio_ekf)/launch/rviz/rviz_euroc_config.rviz"/>

</launch>
